client/src/components/QuestionsAndAnswers/components/QuestionAndAnswers.css

```diff
@@ -50,6 +51,72 @@
   transition: all ease 0.3s;
 }
 
-
 /* SearchBar Basic CSS */
+.searchBarWrapper {
+  position: relative;
+  display: flex;
+  min-width: 100px;
+}
+
+.fa {
+  position: absolute;
+  top: 7px;
+  left: 8px;
+  width: 14px;
+}
+
+.search {
```

Comment:
@HuijunLu Let's use class names more specific to the widget to avoid conflicts. (`search`)